modules/schematics/src/effect/index.ts

```diff
@@ -93,6 +93,21 @@ function addImportToNgModule(options: EffectOptions): Rule {
   };
 }
 
+function getEffectMethod(effectCreator?: boolean) {
+  return effectCreator ? 'createEffect' : 'Effect';
+}
+
+function getEffectStart(name: string, effectCreator?: boolean): string {
+  const effectName = stringUtils.classify(name);
+  return effectCreator
+    ? `load${effectName}s$ = createEffect(() => this.actions$.pipe(`
+    : '@Effect()\n' + `\tload${effectName}s$ = this.actions$.pipe(`;
```

Comment:
The tab here isn't needed, currently it will output:

```ts
  @Effect()
        loadFoos$ = this.actions$.pipe(
    ofType(FooActionTypes.LoadFoos),
    /** An EMPTY observable only emits completion. Replace with your own observable API request */
    concatMap(() => EMPTY)
  );
```

Reply:
hmm weird when I test it I am getting different result 🤔

Reply:
apparently because I use tab as 2 spaces in my pc it looked good...
thanks for your notice!! :)